addons/calendar/models/calendar_event.py

```diff
@@ -975,6 +975,10 @@ def action_mass_archive(self, recurrence_update_setting):
     # MAILING
     # ------------------------------------------------------------
 
+    def _get_responsible(self):
+        self.ensure_one()
+        return self.user_id
+
```

Comment:
Hello! :wave: 
I understood your use case. Culd you please add a description stating that this is an overridable method?
It will help us debugging it in the future :pray:
Thanks!